tests/test_resource_upload.py

```diff
@@ -255,135 +119,40 @@ def test_upload_resources_no_existing_metadata_file(practice_filesystem,
                          )
     stubber.activate()
     mocker.patch('boto3.client', lambda service: s3_client)
-
+    url_prefix = 'prefix/s3'
```

Comment:
This value confused me for a bit as it's misleading. Maybe have it be `https://domain.org/prefix/s3` so the final output reflects `href` values that look like URLs.